File: JavaScript/Lv_0/접미사인지 확인하기.js
function solution(my_string, is_suffix) {
    // const len = my_string.length;
    // for(let i=0; i<len; i++)
    //     {
    //         if(my_string == is_suffix) return 1;
    //         my_string = my_string.substr(1);
    //     }
    // return 0;
    return my_string.endsWith(is_suffix) ? 1 : 0;
}

File: C#/Lv_0/접미사인지 확인하기.cs
using System;

public class Solution {
    public int solution(string my_string, string is_suffix) {
        int len = my_string.Length;
        for(int i=0; i<len; i++)
        {
            if(my_string == is_suffix) return 1;
            my_string = my_string.Substring(1);
        }
        return 0;
    }
}